### ISSUE TITLE
nonprinting checker in production

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#2825]
Error happening again.

![image](https://user-images.githubusercontent.com/47402809/85471155-47452d00-b56d-11ea-8a2b-0563c31f9295.png)

The funny thing is that I was able to create 2019.003 just fine.

I don't see any improper data in the form that might cause it not to save. My session has not timed out and this is not a duplicate accession number.

_Originally posted by @Nicole-Ridgwell-NMMNHS in https://github.com/ArctosDB/data-migration/issues/55#issuecomment-648458721_

**Comment by dustymc** [comment]
I think tomcat is rejecting the characters we're trying to strip; probably need to POST.

**Comment by dustymc** [comment]
@Nicole-Ridgwell-NMMNHS can you clear your cache and try again? This works fine in test and I can't see anything different in the environment, maybe you're running old code?? In any case it will be POSTing in the next release.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I did clear my cache and it worked this time.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I spoke too soon - I was able to create the accession but got the error again when trying to add and save new information to the accession.

![image](https://user-images.githubusercontent.com/47402809/85577917-e3ffdd00-b5f6-11ea-9c78-5dfc63a0e648.png)

**Comment by dustymc** [comment]
OK, it should work in next release. The problem is the linebreaks.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
When is the next release?

**Comment by dustymc** [comment]
I'm waiting on testing, which seems to be happening now - hopefully tonight....

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Ok - thanks!

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Now I get a prompt that suggests I use <br> for line breaks. I followed the suggestion and was able to save. Is this the permanent solution? Having to use html code in data entry is annoying.

**Comment by dustymc** [comment]
This should be a new issue. There have been multiple discussions on this, but I can't find any of them.

The consensus has consistently been that we should prevent nonprinting characters globally, but failing to do so won't break Arctos. It will break things that try to talk to Arctos, and generally act as a denormalizer.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Huh, I didn't know that. I will think about it and do a separate issue if I still think it is important. I would close the issue again, but it appears I don't have that power.
